lambda/src/main/scala/pricemigrationengine/handlers/AmendmentHandler.scala

```diff
@@ -84,17 +86,29 @@ object AmendmentHandler extends CohortHandler {
       invoicePreviewBeforeUpdate <-
         Zuora.fetchInvoicePreview(subscriptionBeforeUpdate.accountId, invoicePreviewTargetDate)
 
-      update <- ZIO.fromEither(
-        ZuoraSubscriptionUpdate
-          .updateOfRatePlansToCurrent(
-            account,
-            catalogue,
-            subscriptionBeforeUpdate,
-            invoicePreviewBeforeUpdate,
-            startDate,
-            PriceCap.priceCorrectionFactor(oldPrice, estimatedNewPrice)
+      update <-
```

Comment:
Are you sure no other cohorts are running. This means every sub will pass through this function?

Reply:
There is a check of whether of not the migration is Membership. If membership we do updateOfRatePlansToCurrentMembership2023 , otherwise we run the legacy code.

Reply:
ah right, i didn't see that if statement. ok nice.